src/ade/assessment_data_export.py

```diff
@@ -94,7 +96,7 @@ def export_jira_data(jira_base_url, jira_credentials_file, jira_filter, xml_file
     # Export XML data from Jira
     try:
         response = requests.get(
-            jira_url, auth=(jira_username, jira_password), verify=False
+            jira_url, auth=(jira_username, jira_password), verify=False  # nosec
```

Comment:
Why is nosec needed here?  It might be good to include a comment.

Reply:
Good catch, I mentioned it in the commit but forgot to add a code comment. Hopefully the comment in 2f19cc6 satisfies.